### DISCUSSION TITLE
很喜欢twitch的表情，想知道有没有大佬能基于betterttv在Blivechat能出个相关的插件

### DISCUSSION THREAD

**Discussion by EdenBush** [xfgryujk/blivechat#208] [想法]
很喜欢twitch的表情，想知道有没有大佬能基于betterttv在Blivechat能出个相关的插件

**Comment by xfgryujk** [comment]
改一下后端配置的`text_emoticon_mappings`就行了

**Reply by EdenBush** [reply]
感谢回复！大佬我是编程小白请原谅我有些地方不懂，如有冒犯请你原谅，我查看了data文件中的config.example配置示例找到了你提到的 `[text_emoticon_mappings]`部分。依据我的理解，配置该部分需要我一个个修改表情映射的触发词和表情图片的URL。这会花费非常多的时间（或许因为我的编程经验不多，理解错误方法）。我查找到了BetterTTV(可以显示Twitch表情网站)的API介绍网址，在此我要再次为我不多的电脑经验抱歉。我想知道是否有方法让Blivechat接入BetterTTV(表情包网站）来实现能在Blivechat显示Twitch表情包的功能。网址如下：https://betterttv.com/developers/api。

**Reply by xfgryujk** [reply]
你写个爬虫把表情填到配置里是最简单的。如果你喜欢折腾也可以改掉这个接口动态获取 https://github.com/xfgryujk/blivechat/blob/e77c26c79206557d379951c22e94e9388f54dda3/api/chat.py#L451

**Reply by EdenBush** [reply]
感谢回复，我会根据你的方法尝试的！
